.golangci.yml

```diff
@@ -36,3 +36,6 @@ issues:
     - linters:
         - staticcheck
       text: "SA1019: l.UseOriginalDst is deprecated: Do not use." # TODO What is the up-to-date alternative ?
+    - linters:
+        - staticcheck
+      text: "c.Http2ProtocolOptions is deprecated: Do not use." # TODO What is the up-to-date alternative ?
```

Comment:
I was pretty sure that comment in the code that ignores it was enough. Did something change with a new version?

Reply:
Yep the messages did appear on my computer so I had to disable it.